starport/cmd/build.go

```diff
@@ -19,14 +19,14 @@ const (
 func NewBuild() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "build",
-		Short: "Build your app",
-		Long: `By default, build command will build your application's binaries
-and add them to your $(go env GOPATH)/bin path.
+		Short: "Build a node binary",
+		Long: `By default, build your node binaries
+and add the binaries to your $(go env GOPATH)/bin path.
 
-If you want to only build binaries for a release, use the --release flag with
---release.targets (optional). Then, binaries built for chosen targets will appear
-in your app's path under release/. You can add any number of targets in GOOS:GOARCH format.
-If you don't provide any, a binary will be created for your own machine.
+To build binaries for a release, use the optional --release flag. The app binaries
```

Comment:
to make it clear, `--release` flag is actually required if you want the `build` command work in the release mode. But `--release.targets` flag is optional when you use `--release`.